### ISSUE TITLE
J4 Privacy Suite / Possibility for a “Do not Track” User Actions

### ISSUE THREAD

**Issue by coolcat-creations** [joomla/joomla-cms#39260]
### Is your feature request related to a problem? Please describe.
In Joomla Privacy Suite, it's either possible to enable User Actions logging and “tracking” or to disable it.
If activated, the user can agree or “stay away”.
It would be good to have a feature as site owner to allow also a third option “do not track me” or “disallow”.

### Describe the solution you'd like
A Setting in the Privacy Suite for Super Users to enable a third possibility for users to opt out of being logged.

**Comment by zero-24** [comment]
Isnt the problem that we have to log whats happening in the system so it can be exported later? Even when not tracked via action logs that data is still in the system but not in a format that can be exported when requested?

**Comment by coolcat-creations** [comment]
You don't have to log, I mean there is a possibility to disable logging?

**Comment by zero-24** [comment]
Yes for sites not falling under GDPR, but I could be wrong too. When you will disable the setting the same data is recorded so the question would be what you export when requested according to GDPR. ;⁠-⁠)

**Comment by zero-24** [comment]
When we are not required it should be disabled by default in order to practice dataminimisation.

**Comment by coolcat-creations** [comment]
Ok, imagine as super admin you want to have the user log on for registered users, but you don't want the logs for super admins. How to solve that?

**Comment by zero-24** [comment]
I do get your point and i'm not against adding such a feature. When its required that we have to collect that data it would be required for super admins too as they can request thier data too. And to check whats happening the actions of an super user is more interesting than a regular user ;⁠-⁠)

When its not legally required that we collect that data an DNT setting can be added not sure what that data is worth than but that for others to decide, if aviable i personally would also set that setting to "Do not track" by default for all users.

But I'm not sure who can tell us whether that collection is required or not.

**Comment by wojtekxtx** [comment]
@zero-24 Data Privacy/Protection lawyer, I suspect

**Comment by coolcat-creations** [comment]
This can be formulated the other 

> And to check whats happening the actions of an super user is more interesting than a regular user ;⁠-⁠)

Yes could be also the other way around: only track superusers and not regular users

**Comment by zero-24** [comment]
> >But I'm not sure who can tell us whether that collection is required or not.
> 
> @zero-24 Data Privacy/Protection lawyer, I suspect

sure yes do you know someone we could ask?

**Comment by zero-24** [comment]
> This can be formulated the other 
> 
> > And to check whats happening the actions of an super user is more interesting than a regular user ;⁠-⁠)
> 
> Yes could be also the other way around: only track superusers and not regular users

Yes i know thats why i asked whether we have to when we have not to log something for every user but still be compliant go and add that option and set it tonnot that by default :⁠-⁠)

**Comment by ReLater** [comment]
When this issue will be accepted:
Perhaps a middle way can be found so that user actions can at least continue to be logged anonymously ("DO-NOT-TRACK-USER"). I, for example, am not one of those people who need to know specifically who changed or did what, but when something was done or whether something was done. For different purposes on different sites.
